Documentation/Visualizations/CompositeBar.md

```diff
@@ -0,0 +1,110 @@
+# Composite Bar Renderer
+
+Workbook allows rendering data using composite bar, a bar made up of multiple bars.
+
+The image below shows the composite bar for database status representing how many servers are online, offline, and in recovering state.
+
+![Image showing a composite bar](../Images/CompositeBar.png)
+
+Composite bar renderer is supported for Grids, Tiles, and Graphs Visualizations
+
+## Adding composite bar renderer
+1. Switch the workbook to edit mode by clicking on the _Edit_ toolbar item.
+2. Use the _Add query_ link.
+3. Set data source to JSON and visualization to Grid.
+4. Add following JSON data
+
+```
+[
+    {"sub":"X", "server": "A", "online": 20, "recovering": 3, "offline": 4, "total": 27},
+    {"sub":"X", "server": "B", "online": 15, "recovering": 8, "offline": 5, "total": 28},
+    {"sub":"Y", "server": "C", "online": 25, "recovering": 4, "offline": 5, "total": 34},
+    {"sub":"Y", "server": "D", "online": 18, "recovering": 6, "offline": 9, "total": 33}
+]
+```
+5. Click on "Column Settings" to open the settings.
+6. Click on "total" column and choose Column renderer as "Composite Bar".
+7. Set following settings under Composite Bar Settings.
+
+![Image showing composite bar column settings](../Images/CompositeBarColumnSettings.png)
+
+8. Add Label: ["online"] of ["total"] are healthy
+9. Set renderer as Hidden for online, offline, and recovering columns (Optional).
+9. Go to Labels and update label for "total" column as "Database Status" (Optional).
+9. Click on "Apply".
+
+
+![Image showing composite bar settings](../Images/CompositeBarSettings.png)
```

Comment:
these images should have text near by, or something calling out why they are here.  something like a `###` header above it that says "the composite bar settings will look like this" and  "the results will look like this"